src/containerDeployment/dockerUtils.ts

```diff
@@ -380,12 +391,25 @@ export async function getDockerPath(executable: string): Promise<string> {
     return "";
 }
 
+/**
+ * Temp fix for the SQL Server 2025 version issue on Mac.
+ * Returns the last working version of SQL Server 2025 for Mac.
+ */
+export function getWorking2025Version(version: string): string {

```

Comment:
better name might be `constructVersionTag()` since this technically handles more than just 2025 (even if the others are just the default case).